phpBB/language/en/email/forum_notify.txt

```diff
@@ -13,8 +12,8 @@ If you want to view the topic, click the following link:
 If you want to view the forum, click the following link:
 {U_FORUM}
 
-If you no longer wish to watch this forum you can either click the "Unsubscribe forum" link found in the forum above, or by clicking the following link:
-
+If you no longer wish to watch this forum click the following link:
 {U_STOP_WATCHING_FORUM}
+Or click the "Unsubscribe forum" link found at the bottom of the forum above.
```

Comment:
What if we would just remove line#17 here? Because then the mail only referrs to links it provides and is not meant to explain basic board functions.

Reply:
I removed those lines completely.